content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+.. _MT_N_layered_Earth:
+
+Response of a layered Earth to a plane wave
+===========================================
+
+.. topic:: Purpose
+
+    We present here a 1D modelisation of the MagnetoTelluric waves in a layered Earth and the associated geophysical data. Our goal is to build better representation of the different physical phenomenons and better understanding of the resulting data.
+
+
+.. raw:: html
+    :file: ./images/movieMT_time.html
+
+Introduction
+------------
+
+We present here a 1D modelisation of the MagnetoTelluric waves in a layered Earth and the associated geophysical data. Our goal is to build better representation of the different physical phenomenons and better understanding of the resulting data. This work follows the derivation presented in :cite:`ward1988` and is supported by interactive apps developed in a `binder`_.
+
+ .. image:: http://mybinder.org/badge.svg 
+    :target: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
+    :align: center
+
+.. _binder: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
+
+MagnetoTelluric is a widely used method, especially for imaging geothermy. Its ability to image deep structure, up to several kilometers depth, is unique in EM geophysics. It is a passive method that use waves generated mostly in the Earth's Atmosphere. High frequencies are mainly produced by lightning strikes all around the globe, travelling through the Earth's Ionosphere that acts as a waveguide. Low Frequencies waves are produced through the interaction of the Earth's Ionosphere with solar wind and Earth's magnetic field.
+
+In MagnetoTellurics problems, the key diagnosed physical property is :ref:`electrical conductivity<electrical_conductivity_index>` :math:`\sigma`, as we expect the contrasts of the others physical properties to be negligible. 
```

Comment:
- "the key **diagnostic**"
- for justifying why we typically ignore the other phys props, I would try to be a bit more specific, something along the lines of "as we expect the impact of other physical properties (magnetic permeability, :math:`\mu` and dielectric permittivity, :math:`\varepsilon`) on the EM responses to be negligible in comparison"